airbyte-db/src/main/java/io/airbyte/db/jdbc/JdbcDatabase.java

```diff
@@ -47,7 +47,47 @@
   void execute(CheckedConsumer<Connection, SQLException> query) throws SQLException;
 
   default void execute(String sql) throws SQLException {
-    execute(connection -> connection.createStatement().execute(sql));
+    try {
+      execute(connection -> connection.createStatement().execute(sql));
+    } catch (SQLException e) {
+      printSQLException(e);
+    }
+  }
+
+  static void printSQLException(SQLException ex) {
```

Comment:
I am worried about swallowing exceptions wholesale like this. Can we instead ensure the problem isn't happening in the query itself?

Reply:
For Oracle no, the problem is not the query. This is to handle statements like `DROP TABLE XPTO` and the table `XPTO` doesn't exist in Oracle.
```
SQLState: 42Y55
Error Code: 30000
Message: 'DROP TABLE' cannot be performed on
'TESTDB.XTPO' because it does not exist.
```

Reply:
I removed this from JdbcDatabase class.